docs/development_suite/api-console/advanced-section/dev-console-config.md

```diff
@@ -61,3 +96,18 @@ This feature is enabled only for `cms-backend` and `v1-adapter` core services. O
 :::warning
 This feature is enabled for all environments. If it is not required to add custom ca certs, for example for a test environment, you must add a secret with an empty `additional-ca.pem` file content.
 :::
+
+## Change the Number of Replicas
+
+To change the number of replicas of a service you need to edit the file in the advanced Section: `core-services.json.` and add the key **replicas** and value the number of replica.
+
+For example, to configure auth service with 5 replicas, the configuration to be provided is as follows:
+
+```json
+"auth-service": {
```

Comment:
`auth-service` is not a service you can configure in this extension 🤔

Reply:
hi @fredmaggiowski what are exactly the services that can be configured from here?

i can add a list to underline this constraint